android/anc/src/main/java/org/smartregister/fhircore/anc/ui/report/ReportScreens.kt

```diff
@@ -273,15 +273,17 @@ fun PatientSelectionBox(
     if (patientSelection.value == ReportViewModel.PatientSelectionType.INDIVIDUAL) {
       Row(modifier = Modifier.padding(start = 24.dp)) {
         Spacer(modifier = Modifier.size(8.dp))
-        SelectedPatientItem(
-          selectedPatient = selectedPatient!!,
-          onCancelSelectedPatient = {
-            onPatientSelectionChange(ReportViewModel.PatientSelectionType.ALL)
-          },
-          onChangeClickListener = {
-            onPatientSelectionChange(ReportViewModel.PatientSelectionType.INDIVIDUAL)
-          }
-        )
+        selectedPatient?.let {
+          SelectedPatientItem(
+            selectedPatient = selectedPatient,
+            onCancelSelectedPatient = {
+              onPatientSelectionChange(ReportViewModel.PatientSelectionType.ALL)
```

Comment:
Can we add a test to cover this line?

Reply:
expected test is written in ReportScreenTest class with name testPatientSelectionChangeListener. tagged you there.
this line would be cover there, will re-check if still not coverings with updating params for this fun